app/ui-message/client/actionButtons/messageAction.ts

```diff
@@ -0,0 +1,22 @@
+import { IUIActionButton } from '@rocket.chat/apps-engine/definition/ui';
+
+import { MessageAction } from '../../../ui-utils/client';
+import { applyButtonFilters } from './lib/applyButtonFilters';
+
+const getIdForActionButton = ({ appId, actionId }: IUIActionButton): string => `${ appId }/${ actionId }`;
+
+// eslint-disable-next-line no-void
+export const onAdded = (button: IUIActionButton): void => void MessageAction.addButton({
+	id: getIdForActionButton(button),
+	icon: 'arrow-loop',
```

Comment:
The icon name expected in this property is not from the same source as the `icon` [in the tabbar/toolbox](https://github.com/RocketChat/Rocket.Chat/blob/feat/add-ui-buttons-to-contexts/app/ui-message/client/actionButtons/tabbar.ts#L11), but it's rather from a much smaller set of icons. Can we unify this? If not, how to better explain that to apps developers?